server/services/ApplicationsDeveloper.js

```diff
@@ -170,15 +170,38 @@ class ApplicationsDeveloper {
   }
 
   async updateKey (data, appId, keyId, keyType, oauthToken, environment) {
-    if (keyType === 'key') {
-      return this.updateRestApiKey(data, appId, keyId, oauthToken, environment);
+    if (keyType === 'key'){
+      return this.updateRestApiKey(data, appId, keyId, oauthToken, environment)
```

Comment:
it might just be the way git is displaying it but it looks like the space between the closing bracket and the open curly bracket was removed for rest key, however the space was kept for stream key and added for web key.